app/assets/stylesheets/base/_tables.scss

```diff
@@ -1,32 +1,27 @@
 table {
-  display: flex;
-  flex-flow: column nowrap;
-  font-size: .8em;
-  margin: 0.5em;
-  line-height: 1.5;
-  flex: 1 1 auto;
+  border-collapse: collapse;
+  margin: 20px auto;
+  width: 85%;
 }
 
-th {
-  font-weight: 700;
+td, th {
+  border: 5px;
+  padding-bottom: 5px;
+  padding-left: 5px;
+  padding-right: 5px;
+  text-align: center;
+  word-wrap: break-word;
 }
 
-th > td {
-  align-items: center;
-  justify-content: center;
-  white-space: normal;
+th {
+  border-bottom: $border;
 }
 
-tr {
-  width: 100%;
+td {
+  border-bottom: $border;
```

Comment:
Line 17 and 21 repeat themselves.
Instead, I would remove these lines and define the same rule on `th, td {}`

Reply:
Done with fixup in ```d1911e9```.

Thanks!